### ISSUE TITLE
error after attempting to import tflearn

### ISSUE THREAD

**Issue by JingjieYang** [tflearn/tflearn#674]
I get the following error after I try to import tflearn in python 3.6:

```(/Users/yangjingjie/anaconda) yangjinjiedeair:MachineLearning yangjingjie$ python main.py 
Traceback (most recent call last):
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/__init__.py", line 61, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 28, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow', fp, pathname, description)
  File "/Users/yangjingjie/anaconda/lib/python3.6/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/Users/yangjingjie/anaconda/lib/python3.6/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: dlopen(/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow.so, 10): Library not loaded: @rpath/libcudart.8.0.dylib
  Referenced from: /Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow.so
  Reason: image not found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 4, in <module>
    import tflearn
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tflearn/__init__.py", line 4, in <module>
    from . import config
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tflearn/config.py", line 3, in <module>
    import tensorflow as tf
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/__init__.py", line 72, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/__init__.py", line 61, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 28, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow', fp, pathname, description)
  File "/Users/yangjingjie/anaconda/lib/python3.6/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/Users/yangjingjie/anaconda/lib/python3.6/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: dlopen(/Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow.so, 10): Library not loaded: @rpath/libcudart.8.0.dylib
  Referenced from: /Users/yangjingjie/anaconda/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow.so
  Reason: image not found


Failed to load the native TensorFlow runtime.

See https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#import_error

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
```

It is possible that I haven't set up other things correctly, but I am surprised to see the reason is actually `image not found`.

**Comment by lipixun** [comment]
Can you import `tensorflow` successfully? It seems an issue of the installation of tensorflow.

**Comment by JingjieYang** [comment]
Yes in fact it is an issue of the installation of tensorflow ... Sorry for my carelessness.

**Comment by dantedevil619** [comment]
I'm having the same error and I can not import tensorflow

**Comment by Isaacnmki** [comment]
Yeah,same but I heard it was fixed in CUDA 8.0
